Doc/whatsnew/3.8.rst

```diff
@@ -698,6 +698,15 @@ Changes in the Python API
   set for regular user accounts.
 
 
+Changes in the C API
+--------------------
+
+* Use of ``#`` variants of formats without ``PY_SSIZE_T_CLEAN`` defined
```

Comment:
Hum, it's non obvious that these formats are used to **parse function arguments**. Maybe mention "ex: PyArg_ParseTuple"?

Reply:
I would add references to PyArg_ParseTuple and Py_BuildValue.

Reply:
Oh right, I always forget about Py_BuildValue() which is the "opposite" or PyArg_ParseTuple().